sentry-samples/sentry-samples-console/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+    java
+    id(Config.QualityPlugins.gradleVersions)
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    implementation(project(":sentry-core"))
+}
+
+val fatJar = task("fatJar", type = Jar::class) {
+    archiveFileName.set("${project.name}-fat.jar")
+    manifest {
+        attributes["Main-Class"] = "io.sentry.samples.console.Main"
+    }
+    from(configurations.runtimeClasspath.get().map { if (it.isDirectory) it else zipTree(it) })
+    with(tasks.jar.get() as CopySpec)
+}
+
+tasks {
+    "build" {
+        dependsOn(fatJar)
+    }
+}
```

Comment:
nothing against doing it, but what's about sparing some bits and using a Gradle plugin? this might break when upgrading Gradle versions and usually plugins react fast to it.
eg https://github.com/johnrengelman/shadow

Reply:
I wanted to avoid extra dependency that's why I don't go with plugin. 

If people build standalone CLI apps then it's common to build fat jars. It only depends if we want this example to show the code or also to actually let users run the sample application.

Reply:
I believe both, but if they run, it'll be thru the IDE anyway, we don't export/allow to download the final jars/apks, it might be a good idea, but for that, we also have our official examples repo.
https://github.com/getsentry/examples

the idea of having samples on this repo is to easier the development, so you can run and debug the SDK and samples easily.

of course also a resource of learning.

that said, I'd go either with the plugin, so we don't need to maintain this piece of code or to not generate a fat jar at all.
thoughts @bruno-garcia ?

Reply:
I am getting convinced that we can just remove fat jar building. It can be always added later if we find there is a need for that.

Reply:
I guess removing then is best, otherwise tbh I'd also avoid the dependency and just leave those lines of code.
If gradle changed and broken I'm sure the plugin would take more time to catch up than us just patching the fix here.

I guess it boils down to how much pain have you had because of dependencies in your life. I avoid them as much as possible

Reply:
@bruno-garcia gradle plugins are not dependencies